build-kmod-framework-laptop-tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -oeux pipefail
+
+cp /tmp/ublue-os-akmods-addons/rpmbuild/SOURCES/_copr_ublue-os-akmods.repo /etc/yum.repos.d/
+
+ARCH="$(rpm -E '%_arch')"
+KERNEL="$(rpm -q "${KERNEL_NAME}" --queryformat '%{VERSION}-%{RELEASE}.%{ARCH}')"
+RELEASE="$(rpm -E '%fedora')"
+
```

Comment:
I think you should take a look at [build-kmod-v4l2loopback.sh](https://github.com/ublue-os/akmods/blob/main/build-kmod-v4l2loopback.sh#L10C1-L13C3) and add a similar conditional exist in this script so that it only runs on main KERNEL_FLAVOR and RELEASE >= 39.

You'll need to move the `cp ` for the COPR repo after the conditional as well.

Reply:
Ok, so the kernel flavor is just nvidia or main right now yea? Given that frameworks don't have nvidia drivers that's safe to exit as well. So I think the != main is still fine. I've got a commit that will do that, but now that I've taken a look at it, I don't think that this is my only concern. Its very likely "a" concern, but there are other things that are still borked I think.

Reply:
Kernel flavors can be found here:  https://github.com/ublue-os/akmods/blob/main/.github/workflows/reusable-build.yml#L30-L34

Reply:
Fantastic, given  that my ```!= main``` should still stand.